src/probeinterface/plotting.py

```diff
@@ -138,6 +139,8 @@ def plot_probe(
         Limits for z dimension
     show_channel_on_click : bool, default: False
         If True, the channel information is shown upon click
+    side : None | "front" | "back"
```

Comment:
I kind of think that I would not know what to expect between front and back or left or right? Both of them seem equally arbitrary, maybe we should type with literal so this surfaces to end users in docstrings more directly.